src/status_im/test_runner.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im.test-runner
```

Comment:
This file is mainly copy of https://github.com/thheller/shadow-cljs/blob/ecb6f9f236b1305fc8cb1092d4cb2dfc2ca12b4a/src/main/shadow/test/node.cljs#L1 with some minor changes below